components/ConversationList/GroupConversationItem.tsx

```diff
@@ -38,9 +38,13 @@ export const GroupConversationItem: FC<GroupConversationItemProps> = ({
   const lastMessagePreview = conversation.lastMessagePreview;
   const { data: groupName } = useGroupNameQuery(userAddress, topic, {
     refetchOnMount: false,
+    staleTime: Infinity,
+    gcTime: Infinity,
   });
   const { data: groupPhoto } = useGroupPhotoQuery(userAddress, topic, {
     refetchOnMount: false,
+    staleTime: Infinity,
+    gcTime: Infinity,
   });
   const { blockGroup } = useGroupConsent(topic);
```

Comment:
`useGroupConsent` will call many other queries and needs to be prevented to query the bridge also?

Reply:
Good catch! I'll update for that too

Reply:
Updated!
Verified it's not being called on the conversation list after initial load any more